### PR TITLE
🔥ON HOLD UNTIL 2.12 RELEASE 🔥 - KUB-885 Added changelog for latest release.

### PR DESCRIPTION
**Pull request by tstec-innovo**

### CODE REVIEW

File: 05.Changelog/01.Changelog-v2.12.2/default.en.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+- 1.14.8
+- 1.15.5
+- 1.16.2
+
+## Major new features:
+
+- Kubernetes 1.16 support was added
+- Removed K8S releases affected by CVE-2019-11253
+- It is now possible to also configure automatic node updates by setting automaticNodeUpdate: true in the updates.yaml. This option implies automatic: true as node versions must not be newer than the version of the corresponding - controlplane.
```
Comment (dionb) [inline 05.Changelog/01.Changelog-v2.12.2/default.en.md]: This is not relevant to customers

File: 05.Changelog/01.Changelog-v2.12.2/default.en.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- Added possibility to add labels to projects and clusters and have these labels inherited by node objects.
+- Added support for Kubernetes audit logging
+- "Connect" button on cluster details will now open Kubernetes Dashboard
+- Pod Security Policies can now be enabled
+- Added support for optional cluster addons
```
Comment (dionb) [inline 05.Changelog/01.Changelog-v2.12.2/default.en.md]: We aren't supporting this for customers yet (as far as I'm aware, I suggest you check with Harald)

File: 05.Changelog/01.Changelog-v2.12.2/default.en.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## Dashboard:
+
+- Kubernetes 1.13 which is end-of-life has been removed.
+- Added Swagger UI for Kubermatic API
```
Comment (dionb) [inline 05.Changelog/01.Changelog-v2.12.2/default.en.md]: We don't want to advertise that we have an API just yet

Comment (dionb) [reply 05.Changelog/01.Changelog-v2.12.2/default.en.md]: Also, I don't think we want to advertise that we use kubermatic

File: 05.Changelog/01.Changelog-v2.12.2/default.en.md (hunk 2 above)
Comment (tstec-innovo) [reply 05.Changelog/01.Changelog-v2.12.2/default.en.md]: These features are already available on dev.imke.cloud, so I assume we're supporting them.

File: 05.Changelog/01.Changelog-v2.12.2/default.en.md (hunk 1 above)
Comment (tstec-innovo) [reply 05.Changelog/01.Changelog-v2.12.2/default.en.md]: You're right about the third point, but I think the first 2 are relevant (ie visible) to customers.

**Pull request merged**